wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -29,6 +24,16 @@
 )
 from .abstract import AbstractRun, AbstractRunner, Status
 
+if "kubernetes" not in sys.modules:
+    msg = "Kubernetes runner requires the kubernetes package. Please install it with `pip install wandb[launch]`."
+    wandb.Error(msg)
+    raise ModuleNotFoundError(msg)
```

Comment:
I think sys.modules only tracks things that have already been successfully imported

Reply:
You could just use `get_module` here right?

Reply:
You mean use `get_module` to make sure kubernetes is installed, then have `from kubernetes import [...]` below like normal? Sure

Reply:
Yeah, I think `sys.module` only contains things that are imported